index.html

```diff
@@ -13,11 +13,81 @@
   </head>
 
   <body>
-    <div class="tooltip m-64">
-      this text has a tooltip
-      <span class="tooltip-text tooltip-text-bottom bg-grey-300"
-        >Tooltip text</span
-      >
+    <div class="p-5">
+      <div class="slideshow">
+        <div class="slides fade">
+          <div class="slide-number">1 / 3</div>
+          <img
+            src="https://picsum.photos/id/1015/500/200"
+            style="width: 100%"
+          />
+          <div class="slide-text">Caption One</div>
+        </div>
+
+        <div class="slides fade">
+          <div class="slide-number">2 / 3</div>
+          <img
+            src="https://picsum.photos/id/1016/500/200"
+            style="width: 100%"
+          />
+          <div class="slide-text">Caption Two</div>
+        </div>
+
+        <div class="slides fade">
+          <div class="slide-number">3 / 3</div>
+          <img
+            src="https://picsum.photos/id/1039/500/200"
+            style="width: 100%"
+          />
+          <div class="slide-text">Caption Three</div>
+        </div>
+
+        <!-- Next and previous buttons -->
+        <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+        <a class="next" onclick="plusSlides(1)">&#10095;</a>
+      </div>
+
+      <!-- The dots/circles -->
+      <div style="text-align: center">
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
+        <span class="dot" onclick="currentSlide(3)"></span>
+      </div>
+
+      <script>
+        var slideIndex = 1;
+        slides(slideIndex);
+
+        // Next/previous controls
+        function plusSlides(n) {
+          slides((slideIndex += n));
+        }
+
+        // Thumbnail image controls
+        function currentSlide(n) {
+          slides((slideIndex = n));
+        }
+
+        function slides(n) {
+          var i;
+          var slides = document.getElementsByClassName("slides");
+          var dots = document.getElementsByClassName("dot");
+          if (n > slides.length) {
+            slideIndex = 1;
+          }
+          if (n < 1) {
+            slideIndex = slides.length;
+          }
+          for (i = 0; i < slides.length; i++) {
+            slides[i].style.display = "none";
+          }
+          for (i = 0; i < dots.length; i++) {
+            dots[i].className = dots[i].className.replace(" active", "");
+          }
+          slides[slideIndex - 1].style.display = "block";
+          dots[slideIndex - 1].className += " active";
+        }
+      </script>
```

Comment:
Inline script?? May be create a JS file and link it in script tag.

Reply:
Thank you for reviewing! I'll keep that in mind.